website/src/pages/guides/successful-monolith-to-microservices-transition.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+
+import Layout from '../../components/Layout'
+import { useCerosScript } from '../../hooks/ceros'
+
+const MonolithToMicroservices: FunctionComponent<PageProps> = props => {
+    useCerosScript()
+
+    return (
+        <Layout
+            location={props.location}
+            meta={{
+                title: '5 key elements of successful monolith-to-microservices migrations',
+                description:
+                    'Employ the five key techniques that the best engineering organizations enable to successfully conduct major architectural transitions.',
+            }}
+            className="bg-white"
+        >
+            <section
+                style={{
+                    position: 'relative',
+                    width: 'auto',
+                    padding: '0 0 286.02%',
+                    height: '0',
+                    top: '0',
+                    left: '0',
+                    bottom: '0',
+                    right: '0',
+                    margin: '0',
+                    border: '0 none',
+                }}
```

Comment:
All these in-line styles appear important for ceros container & iframe to load properly to their sizes. It'll take manual parsing into JSX for devs. This may cause some issues with our lint with the new repo since we discourage style attributes. 🤔

Reply:
Really good point, Becca. If these values are reliable, what do you think of abstracting these into a model that we maintain separately and pass in as an array?

Reply:
Yea I think that's a great idea. I'd like to see what the next Ceros embed is, then make a reusable component for Ceros once we can see which style values change more, if that sounds okay with the team

Reply:
That sounds like a good approach, keeping in mind what might change. Let's go with that plan.

Reply:
I agree! Now that this is coming into fruition, depending what future embed scripts include, I think we could make some common components like `CerosTemplate` or `CerosContent` as [mentioned](https://github.com/sourcegraph/about/pull/5282#discussion_r864073406) previously.